composer.json

```diff
@@ -101,12 +101,14 @@
     "extra": {
         "symfony": {
             "allow-contrib": true,
-            "require": "4.3.*"
+            "require": "4.4.*"
         }
     },
     "scripts": {
         "auto-scripts": {
-            "security-checker security:check": "script"
+            "security-checker security:check": "script",
+            "cache:clear": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
What is the purpose of this command?

Reply:
it got auto-added with the composer install, looks like auto-checker for dependencies:
https://github.com/sensiolabs/security-checker